Wrappers/Python/CMakeLists.txt

```diff
@@ -1,47 +1,16 @@
-#  Copyright 2019 United Kingdom Research and Innovation
-#  Copyright 2019 The University of Manchester
-#
-#  Licensed under the Apache License, Version 2.0 (the "License");
-#  you may not use this file except in compliance with the License.
-#  You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#  See the License for the specific language governing permissions and
-#  limitations under the License.
-#
-# Authors:
-# CIL Developers, listed at: https://github.com/TomographicImaging/CIL/blob/master/NOTICE.txt
-
 option (BUILD_PYTHON_WRAPPER "Build Python Wrapper" ON)
 
 if (BUILD_PYTHON_WRAPPER)
   find_package(PythonInterp REQUIRED)
```

Comment:
Is this necessary?

Reply:
Perhaps not, given that I can't find any usages of `BUILD_PYTHON_WRAPPER(:BOOL)=OFF` on GitHub.
Would probably remove it in a different PR though.